### DISCUSSION TITLE
Is there a way to understand the apple purchase is canceled or renewed?

### DISCUSSION THREAD

**Discussion by andeeDev** [imdhemy/laravel-in-app-purchases#81] [Q&A]
As I understand from the apple documentation on cancel and on renew will be one event `DidChangeRenewalStatus` , but could you make it clear with this package, how can I distinguish whether it is canceled or renewed? 
Thank you for help

**Comment by dev-rizwanqureshi** [comment]
Hi @andeeDev did you find any solution for retrieving whether the subscription is cancelled or not?

**Comment by ervivek2512** [comment]
There is a cancellation date. you can check the rawBody
